CinemaApp/frontend/src/components/PaymentPage.jsx

```diff
@@ -1,44 +1,123 @@
+    onChangeCardNumber(e) {
+        this.setState({
+            cardNumber: e.target.value
+        });
+        console.log("CN changed");
+    }
+    onChangeExpDate(e) {
+        this.setState({
+            expDate: e.target.value
+        });
+        console.log("EXP changed");
+    }
+    onChangeCvc(e) {
+        this.setState({
+            cvc: e.target.value
+        });
+        console.log("CVC Changed");
+
+    }
+
+    onSubmit(e) {
+        e.preventDefault();
+        const payment = {
+            firstName: this.state.firstName,
+            lastName: this.state.lastName,
+            cardNumber: this.state.cardNumber,
+            expDate: this.state.expDate,
+            cvc: this.state.cvc
+        }
+        console.log(payment);
+        window.alert(JSON.stringify(payment));
+
+        axios.post('http://localhost:4000/payments/post', payment)
+        .then(res => console.log(res.data));
+
+    }
+
+    render() {
+        return (
+            <>
+                <div>
+                    <form  id="payment_form" onSubmit={this.onSubmit}>
+                        <div id="left_col">
+                            <h3>Payment Information</h3>
                             <ul className="hidePoint">
                                 <li>
-                                    <label>First Name</label>
-                                    <input type="text" id="first_name" placeholder="Jane"></input>
+                                    <label>Card Number</label>
+                                    <input type="text" value={this.state.cardNumber} onChange={this.onChangeCardNumber} id="card_number" placeholder="1111-2222-3333-4444"></input>
                                 </li>
                                 <li>
-                                    <label>Last Name</label>
-                                    <input type="text" id="last_name" placeholder="Doe"></input>
+                                    <label>Expiry Date</label>
+                                    <input type="text" value={this.state.expDate} onChange={this.onChangeExpDate} id="expiry_date" placeholder="1804 - 18th April"></input>
+                                </li>
+                                <li>
+                                    <label>CVC</label>
+                                    <input type="text" value={this.state.cvc} onChange={this.onChangeCvc} id="cvc" placeholder="739"></input>
                                 </li>
                             </ul>
-                        </form>
-                    </div>
-                    <div id="right_col">
-                        <h3>Payment Information</h3>
-                        <form>
+                        </div>
+                        <div id="right_col">
+                            <h3>Billing Information</h3>
                             <ul className="hidePoint">
                                 <li>
-                                    <label>Card Number</label>
```

Comment:
I assume these will be added back once you've got the routes/model/axios ready for them? If you need any support there let me know :)